libkbfs/bare_root_metadata.go

```diff
@@ -271,6 +271,12 @@ func (md *BareRootMetadataV2) DeepCopy(codec Codec) (BareRootMetadata, error) {
 	return &newMd, nil
 }
 
+// MakeSuccessor implements the ImmutableBareRootMetadata interface for BareRootMetadataV2.
+func (md *BareRootMetadataV2) MakeSuccessor(codec Codec) (BareRootMetadata, error) {
+	// MDv3 TODO: Make a v3 successor.
+	return md.DeepCopy(codec)
```

Comment:
Sounds good to me. I guess that differs from DeepCopy in that it may give a higher-versioned copy?